sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -1041,11 +1041,11 @@ class PlanParserSuite extends AnalysisTest {
         |FROM testData
       """.stripMargin,
       ScriptTransformation(
-        Seq('a, 'b, 'c),
+        Seq(UnresolvedStar(None)),
```

Comment:
Why do we need to update this?

Reply:
> Why do we need to update this?

Since in parser level, we pass UnresolvedStar to replace all output of child's output and then resolve this in Analyzer level to realize supporting aggregation with transform